generated/nidaqmx/_task_modules/channels/ci_channel.py

```diff
@@ -11746,8 +11896,8 @@ def ci_velocity_a_input_dig_fltr_enable(self):
 
         return val.value
 
-    @ci_velocity_a_input_dig_fltr_enable.setter
-    def ci_velocity_a_input_dig_fltr_enable(self, val):
+    @ci_velocity_encoder_a_input_dig_fltr_enable.setter
```

Comment:
these are fine, but add deprecations

Reply:
Added deprecation for the `ci_velocity_a_input_dig_fltr_enable`